app/kumactl/cmd/install/install_tracing.go

```diff
@@ -0,0 +1,58 @@
+package install
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/Kong/kuma/app/kumactl/pkg/install/data"
+	"github.com/Kong/kuma/app/kumactl/pkg/install/k8s"
+	"github.com/Kong/kuma/app/kumactl/pkg/install/k8s/tracing"
+)
+
+type tracingTemplateArgs struct {
+	Namespace string
+}
+
+func newInstallTracing() *cobra.Command {
+	args := struct {
+		Namespace string
+	}{
+		Namespace: "kuma-tracing",
+	}
+	cmd := &cobra.Command{
+		Use:   "tracing",
+		Short: "Install Tracing backend in Kubernetes cluster (Jaeger)",
+		Long:  `Install Tracing backend in Kubernetes cluster (Jaeger) in a 'kuma-tracing' namespace`,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			templateArgs := tracingTemplateArgs{
+				Namespace: args.Namespace,
+			}
+
+			templateFiles, err := data.ReadFiles(tracing.Templates)
+			if err != nil {
+				return errors.Wrap(err, "Failed to read template files")
+			}
+			yamlTemplateFiles := templateFiles.Filter(func(file data.File) bool {
+				return strings.HasSuffix(file.Name, ".yaml")
+			})
+
+			renderedFiles, err := renderFiles(yamlTemplateFiles, templateArgs, simpleTemplateRenderer)
```

Comment:
You should be able to just pass `templateFiles` instead of `yamlTemplateFiles` because all files for tracing are YAML files

Reply:
I used the existing code in `install_metrics.go` that also uses `yamlTemplateFiles`. Should it change in both?

Reply:
It's needed for metrics because we also have .json files, so we have to filter those.

Reply:
Done